### DISCUSSION TITLE
How to use Notification CRDs from /v1beta2

### DISCUSSION THREAD

**Discussion by Freshchris01** [fluxcd/flux2#3498] [Q&A] [answered]
Hello community.
I would like to test the git commit status notification provider provided in the `providers.notification.toolkit.fluxcd.io/v1betav2` CRDs.
But when I check `kubectl api-resources`, I can see that v1betav1 was installed:

![image](https://user-images.githubusercontent.com/3072291/213444087-52596639-993b-47b5-85c8-35fd03504c4d.png)
Is there a way to uptate the flux installation to use betav2 for these resources? Thanks for any help.

**Comment by makkes** [comment, marked as answer]
The notification.toolkit.fluxcd.io/v1beta2 API is available starting with Flux 0.38 so upgrading to that version will also make v1beta2 available on your cluster. An upgrade guide is available [in the Flux docs](https://fluxcd.io/flux/installation/#upgrade).

**Reply by Freshchris01** [reply]
Thanks that worked 👍
